plugins/catalog-backend-module-ldap/src/ldap/read.ts

```diff
@@ -70,6 +71,13 @@ export async function defaultUserTransformer(
   mapStringAttr(entry, vendor, map.name, v => {
     entity.metadata.name = v;
   });
+
+  if (!entity.metadata.name) {
```

Comment:
@JessicaJHee can we add a test for this new behavior too? :pray:

Reply:
Done! I've also added the same check for the group transformer as well 👍

Reply:
@benjdlambert do you mind taking another look at this? Thanks so much in advance :)